lib/data_directory.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+const normalizeFilePath = filePath => {
```

Comment:
This expands the home directory if it contains a tilde.

This is copied from the [`untildify` javascript library](https://github.com/sindresorhus/untildify/blob/master/index.js#L11).